dashboard/app/assets/javascripts/small_footer.js.jsx

```diff
@@ -1,4 +1,4 @@
-/* $ */
```

Comment:
Nope, I added it as a global. While doing that i removed "global $" from a bunch of places. From this file, I had apparently only removed global - at which point this comment wasnt actually accomplishing anything